### ISSUE TITLE
Provenance not included in score?

### ISSUE THREAD

**Issue by dsherret** [jsr-io/jsr#338]
I got 100% somehow.

![image](https://github.com/jsr-io/jsr/assets/1609021/e2816213-d482-4806-9cb9-5023850584f6)

**Comment by lucacasonato** [comment]
Yes, you can currently have either "Provenance" or "Multi runtime support" to get 100% - probably we'll tweak this soon.
